dts/bindings/sensor/ti,tmp112.yaml

```diff
@@ -8,3 +8,21 @@ description: |
 compatible: "ti,tmp112"
 
 include: i2c-device.yaml
+
+properties:
+  sampling-rate:
+    description: Sampling rate in mHz (milli-Hertz)
+    type: int
+    default: 4000
+    enum:
+      - 250
+      - 1000
+      - 4000
+      - 8000
+  full-scale:
```

Comment:
this property is being treated as a boolean (stored in `extended_mode`, which is `bool extended_mode: 1`). DT binding should reflect that as well.